src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String WELCOME_MESSAGE = "Greetings! I'm Blue, your personal chatbot assistant. How may I be of service?";
+    private static final String GOODBYE_MESSAGE = "Till we meet again.";
+    public static void talk(String line) {
+        // no formatting for now
+        System.out.println(line);
+    }
+    private static void warnInvalidInput() {
+        talk("I'm sorry. I'm afraid I can't do that.");
+    }
+    private static void handleLine(String line) {
+        TaskManager tmMaster = new TaskManager();
+        String request = line.split(" ")[0];
```

Comment:
You can consider doing some string processing such as .toLowerCase() to make your input handler more robust.